test-lit/lit.cfg.py

```diff
@@ -53,32 +37,32 @@
 config.test_exec_root = os.path.join(config.flang_obj_root, 'test-lit')
 
 config.substitutions.append(('%PATH%', config.environment['PATH']))
-config.substitutions.append(('%shlibext', config.llvm_shlib_ext))
 
 llvm_config.use_default_substitutions()
 
 # excludes: A list of directories to exclude from the testsuite. The 'Inputs'
 # subdirectories contain auxiliary inputs for various tests in their parent
 # directories.
-config.excludes = ['Inputs', 'CMakeLists.txt', 'README.txt', 'LICENSE.txt', 'debuginfo-tests']
+config.excludes = ['CMakeLists.txt', 'README.txt', 'LICENSE.txt']
```

Comment:
I think we should keep Inputs in this list. Sure we don't need it for any tests in the initial patch, but it is a well-understood feature of LLVM lit tests to have this directory so we should support it. Plus, your comment still talks about it despite it being removed from the list. Agree with removing debuginfo-tests.

Reply:
Done!